libs/designer/src/lib/ui/menuItems/runAfterMenuItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { MenuItem } from '@fluentui/react-components';
+import { bundleIcon, Copy24Filled, Copy24Regular } from '@fluentui/react-icons';
+import { LogEntryLevel, LoggerService } from '@microsoft/logic-apps-shared';
+import { useCallback } from 'react';
+import { useIntl } from 'react-intl';
+
+const RunAfterIcon = bundleIcon(Copy24Filled, Copy24Regular);
+
+export interface RunAfterMenuItemProps {
+  onClick: (e: unknown) => void;
+  isTrigger?: boolean;
+  isScope?: boolean;
+  showKey?: boolean;
+}
+
+export const RunAfterMenuItem = (props: RunAfterMenuItemProps) => {
+  const { onClick, isTrigger = false, isScope = false } = props;
+
+  const intl = useIntl();
+
+  const runAfterMessage = intl.formatMessage({
+    defaultMessage: 'Run After',
+    id: 'rioocq',
+    description: 'Run After text',
```

Comment:
please provide a better description. The description is used to provide better contextual localizations. This description provides no more information than the string itself.

Reply:
Fixed.